src/main/scala/com/dwolla/cloudflare/DnsRecordClient.scala

```diff
@@ -38,7 +38,7 @@ class DnsRecordClient(executor: CloudflareApiExecutor)(implicit val ec: Executio
     }.map((_, record.zoneId))
   }
 
-  def getExistingDnsRecord(name: String, content: Option[String]=None, recordType: Option[String]=None): Future[Option[IdentifiedDnsRecord]] = {
+  def getExistingDnsRecord(name: String, content: Option[String]=None, recordType: Option[String] = None): Future[Option[IdentifiedDnsRecord]] = {
```

Comment:
Not a big deal, but inconsistent spacing on ` = ` for the defaults

Reply:
Addressed.